ow_power_system/include/PowerSystemNode.h

```diff
@@ -12,76 +22,42 @@
-  bool initPrognoser();
-  bool initTopics();
-  void jointStatesCb(const sensor_msgs::JointStateConstPtr& msg);
   double generateTemperatureEstimate();
   double generateVoltageEstimate();
-  void injectFault(const std::string& power_fault_name,
-                   bool& fault_activated,
-                   double& power,
-                   double& voltage,
-                   double& temperature);
-  void injectCustomFault(bool& fault_activated,
-                         const PrognoserVector& sequence,
-                         size_t& index,
-                         double& wattage,
-                         double& voltage,
-                         double& temperature);
-  void injectFaults(double& power, double& temperature, double& voltage);
-  PrognoserMap composePrognoserData(double power,
-                                    double voltage,
-                                    double temperature);
-  void parseEoD_Event(const ProgEvent& eod_event,
-                      owl_msgs::BatteryStateOfCharge& battery_soc_msg,
-                      owl_msgs::BatteryRemainingUsefulLife& battery_rul_msg,
-                      owl_msgs::BatteryTemperature& battery_temperature_msg);
-  void runPrognoser(double electrical_power);
-
-  ros::NodeHandle m_nh;                               // Node Handle Initialization
-  ros::Publisher m_mechanical_power_raw_pub;          // Mechanical Power Raw
-  ros::Publisher m_mechanical_power_avg_pub;          // Mechanical Power Averaged
-  ros::Publisher m_battery_state_of_charge_pub;       // State of Charge Publisher
-  ros::Publisher m_battery_remaining_useful_life_pub; // Remaining Useful Life Publisher
-  ros::Publisher m_battery_temperature_pub;           // Battery Temperature Publisher
-  ros::Subscriber m_joint_states_sub;                 // Mechanical Power Subscriber
-
-  int m_moving_average_window = 25;
-  std::vector<double> m_power_values;
-  size_t m_power_values_index = 0;
-
-  std::unique_ptr<PCOE::Prognoser> m_prognoser;  // Prognoser initialization
+  void applyValueMods(double& power, double& voltage, double& temperature);
+
+  double runPrognoser(double electrical_power);
 
   std::chrono::time_point<std::chrono::system_clock> m_init_time;
 
   // Main system configuration: these values are overriden by values
   // in ../config/system.cfg.
 
-  double m_initial_power = 0.0;         // This is probably always zero
-  double m_initial_temperature = 20.0;  // 20.0 deg. C
-  double m_initial_voltage = 4.1;       // Volts
   double m_min_temperature = 17.5;      // minimum temp = 17.5 deg. C
   double m_max_temperature = 21.5;      // maximum temp = 21.5 deg. C
   double m_battery_lifetime = 2738.0;   // Estimate of battery lifetime (seconds)
   double m_base_voltage = 3.2;          // [V] estimate
   double m_voltage_range = 0.1;         // [V]
   double m_efficiency = 0.9;            // default 90% efficiency
```

Comment:
A lot of the header variables essentially contain filler values as you mentioned. That was simply how they were set up before I started work on this. I'll go ahead and remove the values, since we wouldn't exactly want to continue if ``system.cfg`` failed to initialize a variable anyways.